content/rc/security/database-security/tls-ssl.md

```diff
@@ -7,81 +7,81 @@ categories: ["RC"]
+    ![View Database](/images/rc/view-db.png#no-click "View Database")
 
-3. Under the **Access Control & Security** subsection, click the **SSL Client Authentication** slider.
+2. Select the **Edit** icon to edit the database. ![Add](/images/rc/icon_edit.png#no-click "Edit")
 
-![SSL Client Authentication](/images/rc/ssl-client-auth.png "SSL Client Authentication")
+    ![Edit Database](/images/rc/edit-db.png#no-click "Edit Database")
 
-4. Decide whether you want to require client authentication (also known as "mutual authentication"). By
-enabling client authentication, only those clients that present a valid certificate will be able to connect. If
-you do not want to require client authentication, skip to **step 8**.
+3. In the **Access Control & Security** section, enable the **SSL Client Authentication** setting.
+
+    ![SSL Client Authentication](/images/rc/ssl-client-auth.png "SSL Client Authentication")
+
+4. Decide whether you want to enforce client authentication (also known as "mutual authentication"). By
+enabling client authentication, only those clients that present a valid certificate will be able to connect. If you do not want to require client authentication, skip to **step 8**.
 
 5. To require client authentication, select **Enforce client authentication**.
 
-![Enforce Client Authentication](/images/rc/enforce-client-auth.png "SSL Client Authentication")
+    ![Enforce Client Authentication](/images/rc/enforce-client-auth.png "SSL Client Authentication")
 
-6. Next, either provide an X.509 certificate containing a public key for your client, or generate one by clicking
-**Generate Client Certificate**.
+6. Next, either provide an X.509 certificate containing a public key for your client or use the 
+**Generate Client Certificate** button to generate one.
 
-7. If you generate your certificate using the admin console, then a download will begin containing the following
-artifacts:
+7. If you generate your certificate using the admin console, then a download will begin containing the following artifacts:
    * `redislabs_user.crt` – the certificate's public key.
    * `redislabs_user_private.key` – the certificate's private key.
    * `redislabs_ca.pem` – the Redis Cloud CA certificate.
 
-8. Click the **Update** button to apply your changes and enable TLS.
+8. Select the **Update** button to apply your changes and enable TLS.
 
-![Cancel / Update](/images/rc/cancel-update.png "Cancel / Update")
+    ![Cancel / Update](/images/rc/cancel-update.png "Cancel / Update")
 
 
 {{<note>}}
 Once you've enabled TLS, all client connections to your database must use TLS. Unencrypted connections
 will no longer be permitted.
 {{</note>}}
 
-## Connecting over TLS
+## Connect over TLS
 
 To connect to a Redis Cloud database over TLS, you will need:
 
 * A Redis client that supports TLS
-* The Redis Cloud CA certificate
+* The Redis Cloud client authentication (CA) certificate
```

Comment:
Here, CA looks like an abbreviation for "client authentication", but it's not. I'd remove the parentheses.

Reply:
Yes; that's intentional.